### ISSUE TITLE
Homepage link to item set instead of navigation item is confusing

### ISSUE THREAD

**Issue by IdaAkiwumi** [religious-ecologies/omeka-s-theme#32]
I'm assuming this is a placeholder, but if not the link currently doesn't work. This is the link from the denominations image on the front page, not the one in the menu.

![IMG_7517](https://user-images.githubusercontent.com/618307/98588832-54ec1c80-229a-11eb-93fa-3fc9ceed629a.PNG)

**Comment by IdaAkiwumi** [comment]
It looks like this might only be happening on my end (probably internet connection). So if this isn't the case please close.

**Comment by kimisgold** [comment]
This is correct, but I can see how this is a confusing view to end up on. Ideally this would be a link to the denominations view, but we don't currently have a means within the Omeka S interface of linking to a page with an associated image. [We might get this in an upcoming feature though](https://github.com/omeka/omeka-s/issues/1637).

**Comment by lmullen** [comment]
In addition to denominations and schedules, the counties block could also go to the browse by locations navigation link instead of an item set.

**Comment by kimisgold** [comment]
We're no longer linking to item sets on the front page, instead directing each of those links to views within the main navigation (locations, denominations, browse schedules).
